data/json/species.json

```diff
@@ -133,6 +133,12 @@
     "footsteps": "shuffling.",
     "bleeds": "fd_blood"
   },
+  {
+    "type": "SPECIES",
+    "id": "NEMESIS",
+    "description": "a zombie",
+    "footsteps": "shuffling."
+  },
```

Comment:
Should it bleed?

Reply:
moot point it turns out! i'm realizing now that the 'nemesis' species isnt actually necessary to the implementation i wound up with, so i think im going to remove this entirely.